apps/hash-frontend/src/components/hooks/use-users-with-links.ts

```diff
@@ -0,0 +1,83 @@
+import { ApolloQueryResult, useQuery } from "@apollo/client";
+import { UserProperties } from "@local/hash-isomorphic-utils/system-types/shared";
+import {
+  AccountId,
+  Entity,
+  EntityRootType,
+  Subgraph,
+} from "@local/hash-subgraph";
+import { getRoots } from "@local/hash-subgraph/stdlib";
+import { useMemo } from "react";
+
+import {
+  QueryEntitiesQuery,
+  QueryEntitiesQueryVariables,
+} from "../../graphql/api-types.gen";
+import { queryEntitiesQuery } from "../../graphql/queries/knowledge/entity.queries";
+import { constructUser, User } from "../../lib/user-and-org";
+
+/**
+ * Retrieves a specific set of useranizations, with their avatars and members populated
```

Comment:
```suggestion
 * Retrieves a specific set of users, with their avatars populated
```

Reply:
useranizations! A find and replace favorite ❤️